driver/src/main/java/org/neo4j/driver/internal/messaging/ValuePacker.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2002-2018 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.driver.internal.messaging;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.neo4j.driver.v1.Value;
+
+public interface ValuePacker
+{
+    void packStructHeader( int size, byte signature ) throws IOException;
+
+    void pack( String string ) throws IOException;
+
+    void pack( Value value ) throws IOException;
```

Comment:
This method seems not used outside `ValuePackerV1`. Maybe not needed to be on the interface?

Reply:
Method is used in `RecordMessageEncoder`